src/java/org/apache/nutch/fetcher/FetcherThreadPublisher.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nutch.fetcher;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.nutch.publisher.NutchPublishers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class handles the publishing of the events to the queue implementation. 
+ *
+ */
+public class FetcherThreadPublisher {
+
+  private static NutchPublishers publisher;
+  private static final Logger LOG = LoggerFactory.getLogger(FetcherThreadPublisher.class);
+
+  /**
+   * Configure all registered publishers
+   * @param conf
```

Comment:
Look at the Javadoc specification for JDK8.
You cannot simply enter @param $paramName, you need to add a description of
the $pqramName after.
There are plenty of examples peppered throughout the code base already.
Good work Sujen.

On Thursday, August 25, 2016, Sujen Shah <notifications@github.com> wrote:

> In src/java/org/apache/nutch/fetcher/FetcherThreadPublisher.java
> <https://github.com/apache/nutch/pull/138#discussion_r76332711>:
>
> > +import org.apache.nutch.publisher.NutchPublishers;
> > +import org.slf4j.Logger;
> > +import org.slf4j.LoggerFactory;
> > +
> > +/**
> > + * This class handles the publishing of the events to the queue implementation.
> > + *
> > + */
> > +public class FetcherThreadPublisher {
> > +
> > +  private static NutchPublishers publisher;
> > +  private static final Logger LOG = LoggerFactory.getLogger(FetcherThreadPublisher.class);
> > +
> > +  /**
> > +   * Configure all registered publishers
> > +   * @param conf
>
> @lewismc <https://github.com/lewismc>, I addressed the other comments in
> the PR, what are you trying to tell me here, I did not understand.
> Thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/nutch/pull/138/files/85b5103eb1d1653fc1207eb1e18cf523607892f8#r76332711>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHJl84hHkvnE34nXsizUEux8YRjWMbbks5qjg_8gaJpZM4Jau45>
> .
>


-- 

*Lewis*
Dr. Lewis J. McGibbney Ph.D, B.Sc
Director
Phone: +1(626)498-3090
Skype: lewis.john.mcgibbney
Email: lewis.mcgibbney@gmail.com